app/controllers/alchemy/pages_controller.rb

```diff
@@ -37,23 +70,39 @@ def sitemap
 
     private
 
-    # Load the current page and store it in @page.
+    def render_fresh_page
+      respond_to do |format|
+        format.html { render action: :show, layout: !request.xhr? }
```

Comment:
Style/BlockDelimiters: Prefer `do...end` over `{...}` for procedural blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)